attractors/clifford_panel.ipynb

```diff
@@ -12,14 +12,12 @@
+    "src='./assets/clifford_screenshot.png'/>\n",
```

Comment:
The reason for embedding the image before was because of finding no way to make the image be found when running Jupyter in git, when building the website, and when running Jupyter on the downloaded notebook. If that's no longer an issue, fine not to embed it any more.